### ISSUE TITLE
SteamLink streaming for Non-Steam Games not working while Revive is installed

### ISSUE THREAD

**Issue by ZC2K** [LibreVR/Revive#531]
Quite not sure if i am the only one with this issue. But if Revive is installed, i can't stream Non-Steam Games from Steam to my SteamLink. Sound is there but no picture, also the Occulus-Background-Process is crushing when streaming starts.
If i uninstall Revive everything works fine.

**Comment by Armada651** [comment]
I have a SteamLink myself and have no problem running Non-Steam games. Are you running both SteamVR and SteamLink streaming at the same time perhaps?

**Comment by ZC2K** [comment]
No. At first i couldn't stream at all or with very low FPS because of TriDef (fake VR). Then everything worked (after removing TriDef) except Non-Steam Games. But if it works for you then i will do some testing, because this means for me that it must work with Revive installed.
